boards/arm/thingy53_nrf5340/thingy53_nrf5340_fem.dts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+ / {
+	nrf_radio_fem: nrf21540_fem {
+		compatible = "nordic,nrf21540-fem";
+		rx-en-gpios = <&gpio1 11 GPIO_ACTIVE_HIGH>;
+		mode-gpios  = <&gpio1 12 GPIO_ACTIVE_HIGH>;
+		pdn-gpios   = <&gpio1 10 GPIO_ACTIVE_HIGH>;
+		tx-en-gpios = <&gpio0 30 GPIO_ACTIVE_HIGH>;
```

Comment:
Should there also be `GPIO_PULL_DOWN` as in net core DTS?

Reply:
You could also make a note in `ug_radio_fem.rst` file saying that for `Thingy:53` FEM is defined by default in DTS (to avoid confusion).
I will check if it works for me asap.

Reply:
> Should there also be GPIO_PULL_DOWN as in net core DTS?

@jciupis Do you know? I've just tried to keep the same GPIO configuration from where I removed it.

> You could also make a note in ug_radio_fem.rst file saying that for Thingy:53 FEM is defined by default in DTS (to avoid confusion).
I will check if it works for me asap.

We would need this change downstream first (since the UG is downstream). Because it is currently broken downstream.

Reply:
While we're talking about downstream, at some point we will want to make the MPSL layer respect the `fem` property now introduced upstream, instead of just relying on the magic `nrf_radio_fem` node label. This will requires some sort of transition period. Who do we need to coordinate this with?

Reply:
@rugeGerritsen ^^

Reply:
@joerchan I'm not sure what the correct configuration for Thingy53 should be. I think it should be the same for both cores though, so maybe @emob-nordic could shed some light on whether `GPIO_PULL_DOWN` is indeed needed in this case.

As for @mbolivar-nordic's comment - @czeslawmakarski I think changes related to FEM representation in devicetree affect us too, right?

Reply:
Other way around, afaik `GPIO_PULL_DOWN` is correct configuration, so `GPIO_PULL_DOWN` should be removed from `thingy53_nrf5340_cpunet.dts` but i would appreciate if @anangl could have a look also. Sorry for the noise.